lib/core/templates.html

```diff
@@ -464,6 +464,19 @@ <h3 class="categoryButton" data-category="{{name}}">{{name}}</h3>
 			</div>
 		</script>
 		<!--/imgurgifvUI-->
+		<!--pornbotUI-->
+		<script id="imgurgifvUI" type="application/x-template">
```

Comment:
Hey thanks for accepting us! If there are any changes you'd like us to make
to our code we'll be happy to fix it up right away.

I believe we hacked up a bit of gfycat's and a bit of imgur's code for
this.

On Thu, Jun 11, 2015 at 1:23 PM, Andy Tuba <notifications@github.com> wrote:

> In lib/core/templates.html
> <https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2272#discussion_r32245406>
> :
>
> > @@ -464,6 +464,19 @@ <h3 class="categoryButton" data-category="{{name}}">{{name}}</h3>
> >  			</div>
> >  		</script>
> >  		<!--/imgurgifvUI-->
> > +		<!--pornbotUI-->
> > +		<script id="imgurgifvUI" type="application/x-template">
>
> we should maybe have a general-use video player instead of copying these
> players over and over.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2272/files#r32245406>
> .
>